theme/themes/pxt/globals/site.variables

```diff
@@ -160,6 +161,17 @@
 @trashIconColor: @primaryColor;
 
 /*-------------------
+   Serial editor
+--------------------*/
+@serialBackgroundColor: lighten(@simulatorBackground, 2);
+@serialTextColor: white;
+@serialGraphBackground: @lightGrey;
```

Comment:
use @darkTextColor instead of white.

Reply:
Doesn't look like @darktextcolor is defined?